### PR TITLE
[SWA-134][IMPROVEMENT] - Fix CoW Trade type errors

### PR DESCRIPTION
**Pull request by ElRodrigote**
## Fixes: [SWA-134](https://linear.app/swaprhq/issue/SWA-134/swapr-sdk-fix-cowtrade-type-errors)

# Description

* Install Prettier and the ESLint plugin to run the linter through the codebase.
* Update CoW SDK trade new types, remove unneeded ESLint skips

### CODE REVIEW

**Review by Diogomartf — APPROVED** [review]

File: src/entities/trades/gnosis-protocol/CoWTrade.ts [hunk 1]
```diff
@@ -26,6 +24,13 @@ import { CoWTradeError } from './CoWTradeError'
 import { CoWTradeGetBestTradeExactInParams, CoWTradeGetBestTradeExactOutParams, CoWTradeParams } from './types'
 
 const ZERO_PERCENT = new Percent(ZERO, ONE)
+enum CowChainId {
+  MAINNET = 1,
+  RINKEBY = 4,
+  GOERLI = 5,
+  GNOSIS = 100,
+  SEPOLIA = 11155111,
+}
```
Comment (berteotti) [inline src/entities/trades/gnosis-protocol/CoWTrade.ts:33]: Is it possible to import this values?

Comment (ElRodrigote) [reply src/entities/trades/gnosis-protocol/CoWTrade.ts:33]: Yes! Already did it! Also added Sepolia to our ChainID constant

Comment (berteotti) [reply src/entities/trades/gnosis-protocol/CoWTrade.ts]: Sorry, I was to vague. Import this values from cow-sdk. They must have their own ChaindId on the SDK

**Review by berteotti — APPROVED** [review]

File: src/entities/trades/gnosis-protocol/CoWTrade.ts (hunk 1 above)
Comment (ElRodrigote) [reply src/entities/trades/gnosis-protocol/CoWTrade.ts]: As we talked offline:
We will keep the current version of the code as CoW SDK provides a ChainID enum that lacks some of the chain IDs needed for this use case.

**Pull request merged**